BaseCollector.py

```diff
@@ -50,11 +51,24 @@ def logout_handles(self):
             print("Logging out from server {}".format(server))
             handle.logout()
 
+    def get_config_data(self, key=None):
+        """
+
+        :return:
+        """
+        with open(self.config) as conf:
+            content = json.load(conf)
```

Comment:
Give it a try except in case it it not there. The configfile itself is provided, but it could be go missing during runtime or even not be there before